.github/workflows/promote-rc.yml

```diff
@@ -26,11 +28,16 @@ jobs:
       - name: "make release/promote-oss/dev-to-rc"
         run: |
           make release/promote-oss/dev-to-rc
+      - name: Update emissary tag in apro
```

Comment:
you can probably make this a different job in the workflow. that way if this step fails we still get RC info and crap in the slack message

Reply:
also this is double true becuase at this point the tree is dirty so RELEASE_VERSION won't be something you like i don't think

Reply:
Good point!